### DISCUSSION TITLE
Display not working - no matter what

### DISCUSSION THREAD

**Discussion by fraxex** [tbnobody/OpenDTU#1072] [General]
Hi everyone,

i bought the following equipment:
- ESP32S CP2102 TYPE-C with ESP-WROOM32
- NRFxxx
and this Display: https://www.az-delivery.de/en/products/1-3zoll-i2c-oled-display

So far, everything works fine, except the Display keeps being black. No matter what.
I reflashed, i load the Pin Mapping json, i changed the pins and i bought a second display. Nothing worked. 
I followed the instruction for the display here on github aswell, but i dont get it running.
What am i doing wrong - am i missing something essential or is it a compatibility issue? 

Thanks in advance!

**Comment by tbnobody** [comment]
Please show is some pictures of your wiring and also the pin assignment in the settings --> hardware view.

**Reply by fraxex** [reply]
Of course, sorry.

![2023-06-24 13_05_12-OpenDTU — Mozilla Firefox](https://github.com/tbnobody/OpenDTU/assets/137612542/a1a304c4-dbfe-4cfc-b368-676b9947b92f)
![2023-06-24 13_05_37-OpenDTU — Mozilla Firefox](https://github.com/tbnobody/OpenDTU/assets/137612542/28ffb036-cceb-4195-bd6c-3cee341b48e5)

I used this as reference:
![2023-06-24 13_09_11-Balkonkraftwerk Teil 8_ OpenDTU](https://github.com/tbnobody/OpenDTU/assets/137612542/33b52a78-5f60-4e2a-a5f8-f5ffb88ac434)

And this is how it looks, i also changed the cable once because i thought they are might be broken or something.
VDD -> 5V
GND -> GND
SCK -> P22
SDA -> P21

![IMG_20230624_130720](https://github.com/tbnobody/OpenDTU/assets/137612542/44ec7f29-b817-47f6-a045-38605bcd675d)
![IMG_20230624_130736](https://github.com/tbnobody/OpenDTU/assets/137612542/370cf488-7e46-4cd2-8daf-399e60609c1a)

![IMG_20230624_130623](https://github.com/tbnobody/OpenDTU/assets/137612542/1d326051-1ca8-462a-9159-a48962466e1a)
![IMG_20230624_130642](https://github.com/tbnobody/OpenDTU/assets/137612542/77dbe13f-8d03-41f2-aaaf-f6ae340f149c)

![IMG_20230624_130609](https://github.com/tbnobody/OpenDTU/assets/137612542/ca5c0b99-ae0b-434a-a3bf-009cd7207019)

**Comment by fraxex** [comment]
Anyone has an idea?
Thanks

**Comment by tbnobody** [comment]
When looking at the images I couldn't find any obvious mistakes.

Have you already tried to use a different power supply? We already had the case that the power supply was too weak to provide the starting power of the ESP and the display and therefor the display wasn't initialized properly.

**Comment by fraxex** [comment]
Yes. I tried four different power supply and an usb-port. Also i tried different pins on the ESP.

**Comment by Eisbier2000** [comment]
I had the same problem. i tried out several esp's and dispalys. The solution was to use the Ground-Pin on the other side. I also changed the pin to 32 (clk) and 33 (data) to have everything on one side. But I don't think this was the solution yet. So just give it a try.

**Reply by huckes** [reply]
@fraxex 
Perhaps the display is dead? 
Did you try to use annother display or did you connect the display to another circuit and it worked there?

**Comment by fraxex** [comment]
Yesterday a second esp arrived and now it's working. Both Displays are fine. My guess, without further checking, the 5V pin on the first esp is broken or something.
The Topic can be closed, thanks to everyone.
